geomesa/src/main/scala/geotrellis/geomesa/geotools/FeatureToGeoMesaSimpleFeatureMethods.scala

```diff
@@ -0,0 +1,11 @@
+package geotrellis.geomesa.geotools
+
+import geotrellis.proj4.CRS
+import geotrellis.util.MethodExtensions
+import geotrellis.vector.{Feature, Geometry}
+import org.opengis.feature.simple.SimpleFeature
+
+trait FeatureToGeoMesaSimpleFeatureMethods[G <: Geometry, T] extends MethodExtensions[Feature[G, T]] {
+  def toSimpleFeature(featureName: String, featureId: Option[String] = Some(null), crs: Option[CRS] = None)(implicit transmute: T => Seq[(String, Any)]): SimpleFeature =
```

Comment:
Oh. Yeah you can comment commits rather than prs, might of been what happened here.